crates/iota-indexer/src/backfill/mod.rs

```diff
@@ -63,13 +74,27 @@ pub enum BackfillKind {
 /// implements the `IngestionBackfill` trait.
 #[derive(ValueEnum, Clone, Debug)]
 #[non_exhaustive]
-pub enum IngestionBackfillKind {}
+pub enum IngestionBackfillKind {
+    TxWrappedOrDeletedObjects,
```

Comment:
Nit: we could add some rust doc , it will be helpful for us and also when using the cli in conjunction with `--help` it will be clearer what it does.